src/test/config.sol

```diff
@@ -1669,6 +1669,36 @@ contract Config {
             calc_cut:     9900,
             lerp:         false
         });
+        afterSpell.collaterals["RWA007-A"] = CollateralValues({
```

Comment:
these latest rwa config blocks (rwa007, rwa008, rwa009) could be grouped above together with the other rwa ilk types, following the current ordering attempt (to put rwa007 before rwa008).

Reply:
Not sure what you mean. The RWA types are already all grouped together.

Reply:
yes, the latest ones, but they're still detached from the previous ones (rwa006, ...)